include/arch/arm/cortex_m/mpu/nxp_mpu.h

```diff
@@ -14,10 +14,12 @@
 #define NXP_MPU_REGION_NUMBER 12
 
 /* Read Attribute */
-#define MPU_REGION_READ  ((1 << 2) | (1 << 8) | (1 << 14))
+#define MPU_REGION_READ  ((1 << 2) | (1 << 8) | (1 << 14) | \
+			  (1 << 18) | (1 << 19) | (1 << 20))
```

Comment:
We should clean up the magic # here.

Seems like you are setting M3UM field, if so should probably be 0x7 << 18, also why is execute (X) needed?